include/exadg/operators/operator_base.cpp

```diff
@@ -2090,9 +2012,13 @@ template<int dim, typename Number, int n_components>
 bool
 OperatorBase<dim, Number, n_components>::evaluate_face_integrals() const
 {
-  return integrator_flags.face_evaluate.do_eval() || integrator_flags.face_integrate.do_eval();
+  auto const eval_anything = dealii::EvaluationFlags::values | dealii::EvaluationFlags::gradients |
+                             dealii::EvaluationFlags::hessians;
+
+  return (integrator_flags.face_evaluate | integrator_flags.face_integrate) & eval_anything;
```

Comment:
Shouldn't it be easier to do
```suggestion
  return integrator_flags.face_evaluate != dealii::EvaluationFlags::nothing;
```

Reply:
why don't you use the variable `face_integrate`?

Reply:
Indeed, we need both, but I would still check whether they are not `nothing`.